CHANGELOG.md

```diff
@@ -86,8 +106,10 @@ DMS is now secured against the [recently published spoofing attack "SMTP Smuggli
   - Debugging - Raise awareness of temporary downtime during certificate renewal that can cause a failure to deliver local mail ([#3718](https://github.com/docker-mailserver/docker-mailserver/pull/3718))
 - **Internal:**
   - Postfix configures `virtual_mailbox_maps` and `virtual_transport` during startup instead of using defaults (configured for Dovecot) via our `main.cf` ([#3681](https://github.com/docker-mailserver/docker-mailserver/pull/3681))
-- **Rspamd:**
-  - Upgraded to version `3.7.5`. This was previously inconsistent between our AMD64 (`3.5`) and ARM64 (`3.4`) images ([#3686](https://github.com/docker-mailserver/docker-mailserver/pull/3686))
+  - Updates to `packages.sh`:
+    - The script now uses `/etc/os-release` to determine the release name of Debian
+    - Removed custom installations of Fail2Ban, getmail6 and Rspamd
+    - Updated packages lists and added comments for maintainability
```

Comment:
Note this changelog addition incorrectly handled resolving the merge conflict / rebase?

Shouldn't be swapping out changelog entries from prior releases :P

Reply:
Was this just the line

```txt
    - Updated packages lists and added comments for maintainability
```

?

Reply:
Hopefully completely resolved in 7cdf318d4dbfa975c6d63eff434dbf8241ed048b